server/service/src/sync/program_schemas/hiv_care_program.json

```diff
@@ -107,8 +107,8 @@
         "mother": {
           "$ref": "#/definitions/HIVRelation"
         },
-        "partnerHivStatus": {
-          "$ref": "#/definitions/HIVPartnerStatus"
+        "partnerHIVStatus": {
```

Comment:
Unrelated, but needed to be changed to match the change @clemens-msupply did yesterday (I think)